_posts/app/2018-02-07-metrics.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Applications Metrics
+modified_at: 2018-02-07 00:00:00
+category: app
+tags: app metrics
+---
+
+We gather a few performance metrics of your application that are accessible on the dashboard of your
+application. Just head to the Metrics tab of your application to see charts of these key metrics.
+
+## Global View
+
+This page contains two main parts: **application charts** and **containers charts**. You can adjust
+the viewing period to 3h, 12h, 24h, 48h and 72h in the past. We try to autodetect your timezone from
+your browser and you can still change it later if needed.
+
+### Application Charts
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_metrics_application.png" %}
+{% include mdl_img.html %}
+
+The application chart displays global data that are not container specific: events and routing
+metrics.
+
+The **events chart** is here to show you all the events that happened during the viewing period.
+This can help you link an application behaviour with events that happened on the platform, e.g. spot
+a deployment that contains a memory leak or follow your application behaviour after a scale up (or
+down).
+
+A lot of events are available on your application timeline but we think that only a few are relevant
+on the metrics view:
+
+- Restart event
+- Deploy event
+- Scale event
+
+You will find the complete list of events in our [developer
+documentation](https://developers.scalingo.com/events.html)).
+
+The **requests per minute** is here to show you how many requests your application gets per minute. It
```

Comment:
`is here to show how many requests are received by the application`

Also try to remove a maximum of `you/your`, it's not really beautiful/formal